src/puppet-web/wechaty-bro.js

```diff
@@ -378,6 +379,19 @@
     })
     return true
   }
+
+  function hookProcess() {
+    var chatFactory = WechatyBro.glue.chatFactory
+    var rootScope = WechatyBro.glue.rootScope
+    // hook chatFactory._recalledMsgProcess, resolve emit RECALLED type msg
+    chatFactory.__recalledMsgProcess = chatFactory._recalledMsgProcess
+    chatFactory._recalledMsgProcess = function (msg) {
+      var _this = this
+      _this.__recalledMsgProcess(msg)
```

Comment:
I modified it, it can work:
```javascript
    chatFactory._recalledMsgProcess = function (msg) {
      chatFactory.__recalledMsgProcess(msg)
      chatFactory.addChatMessage(msg)
    }
```